src/js/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -1,10 +1,46 @@
+import { Link } from 'react-router';
 import React from 'react';
 
 class FacilityLocatorApp extends React.Component {
+  renderBreadcrumbs() {
```

Comment:
Usually the breadcrumbs are generated by metalsmith so I think we need to decide what the breadcrumbs should say and if they should be generated by React or by metalsmith.